### PR TITLE
chore: update explanation for modes in Timescale Console.

### LINKED ISSUE #3473: [Docs RFC] PopSQL updates
Integrate comments from https://github.com/timescale/docs/pull/3451 into the docs.

### PR DESCRIPTION
**Pull request by billy-the-fish**

### CODE REVIEW

File: _partials/_cloud-connect.md [hunk 1]
```diff
@@ -12,16 +12,16 @@ To connect to a service:
 1. **Connect to your service**
 
     Use either:  
-    - [Data][popsql] in Timescale Console:  
-      1. In [`Data` in Timescale Console][portal-data-mode], select a service and enter your password, then click `Connect`.
+    - [Data mode][popsql] in Timescale Console:  
+      1. In [data mode][portal-data-mode] in Timescale Console, select a service and enter your password, then click `Connect`.
```
Comment (atovpeko) [inline _partials/_cloud-connect.md]:
```suggestion
      1. In the [data mode][portal-data-mode] in Timescale Console, select a service and enter your password, then click **Connect**.
```

File: _partials/_cloud-connect.md [hunk 2]
```diff
@@ -12,16 +12,16 @@ To connect to a service:
...
       
          You find your password in the config file you just downloaded.
       2. Select a query to edit:
          ![Select a query to edit](https://assets.timescale.com/docs/images/data-mode-query-window.png)
 
     - [SQL editor][run-sqleditor] in Timescale Console:
     
-       In [`Ops` in Timescale Console][portal-ops-mode], select a service, then click `SQL editor`. 
+       In [ops mode][portal-ops-mode] in Timescale Console, select a service, then click `SQL editor`.
```
Comment (atovpeko) [inline _partials/_cloud-connect.md]:
```suggestion
       In the [ops mode][portal-ops-mode] in Timescale Console, select a service, then click **SQL editor**. 
```

File: _partials/_cloud-connect.md [hunk 3]
```diff
@@ -34,12 +34,13 @@ To connect to a service:
...
 
-   You can now run queries for this service.  
-
+   You can now run queries for this service.
 
 Quick recap. You:
-- Manage your services in the [`Ops` in Timescale Console][portal-ops-mode]
-- Manage your data in [`Data` in Timescale Console][portal-data-mode]
+- Manage your services in the [ops mode][portal-ops-mode] in Timescale Console:  add read replicas and enable 
+  high-availability, configure compression, change parameters and so on.
```
Comment (atovpeko) [inline _partials/_cloud-connect.md]:
```suggestion
  high-availability, configure compression, change parameters, and so on.
```

File: _partials/_cloud-connect.md [hunk 4]
```diff
@@ -34,12 +34,13 @@ To connect to a service:
...
+- Analyze your data in [data mode][portal-data-mode] in Timescale Console: write queries with
```
Comment (atovpeko) [inline _partials/_cloud-connect.md]:
```suggestion
- Analyze your data in the [data mode][portal-data-mode] in Timescale Console: write queries with
```

File: _partials/_service-overview.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  <tr>
+    <th align="center">Ops mode</th>
+    <th align="center">Data mode</th>
+  </tr></thead>
+<tbody>
+  <tr>
+    <td >
+    <img class="main-content__illustration"
+    src="https://assets.timescale.com/docs/images/ops-mode-overview.png"
+    alt="Timescale Console data mode"/>
```
Comment (atovpeko) [inline _partials/_service-overview.md]:
```suggestion
    alt="Timescale Console ops mode"/>
```

File: _partials/_service-overview.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    <td >
+    <img class="main-content__illustration"
+    src="https://assets.timescale.com/docs/images/data-mode-overview.png"
+    alt="Timescale Console data mode"/>
+</td>
+
+  </tr>
+  <tr>
+    <td >
+Perform sysops on your services:
```
Comment (atovpeko) [inline _partials/_service-overview.md]:
```suggestion
You use the ops mode to:
```

File: _partials/_service-overview.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+<ul>
+<li>Ensure data security with high availability and read replicas</li>
+<li>Save money using compressed and tiered storage</li>
+<li>Enable PostgreSQL extensions to add extra functionality</li>
+<li>Increase security using VPCs </li>
+<li>Perform day-to-day administration</li> 
+</ul>
+</td>
+    <td >
+Powered by PopSQL, you use Data mode to:
```
Comment (atovpeko) [inline _partials/_service-overview.md]:
```suggestion
Powered by PopSQL, you use the data mode to:
```

File: getting-started/run-queries-from-console.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Run your queries from Timescale Console
-excerpt: Use Data mode or SQL editor to run SQL queries, create charts and dashboards, and collaborate with teammates.
+excerpt: Use data mode or SQL editor to run SQL queries, create charts and dashboards, and collaborate with teammates.
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
excerpt: Use the data mode or SQL editor to run SQL queries, create charts and dashboards, and collaborate with teammates.
```

File: getting-started/run-queries-from-console.md [hunk 9]
```diff
@@ -9,38 +9,42 @@ content_group: Getting started
...
-- [Data mode][run-popsql]: a comprehensive UI that you use to write queries, visualize data, 
-   and share your results.
-- [SQL editor][run-sqleditor]: a simple UI to run queries for each individual service.
+In Timescale Console you can use the following ways to run SQL queries against your database:
 
+- [Data mode][run-popsql]: a rich experience powered by PopSQL. You can write queries with 
+  autocomplete, save them in folders, share them, create charts/dashboards, and much more.
 
-If you prefer the command line to a Cloud SQL editor, use [psql][install-psql].
+- [SQL editor][run-sqleditor]: a simple UI in ops mode that you use to run ad-hoc ephemeral
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
- [SQL editor][run-sqleditor]: a simple UI in the ops mode that you use to run ad-hoc ephemeral 
```

File: getting-started/run-queries-from-console.md [hunk 10]
```diff
@@ -9,38 +9,42 @@ content_group: Getting started
...
-- [SQL editor][run-sqleditor]: a simple UI to run queries for each individual service.
+In Timescale Console you can use the following ways to run SQL queries against your database:
 
+- [Data mode][run-popsql]: a rich experience powered by PopSQL. You can write queries with 
+  autocomplete, save them in folders, share them, create charts/dashboards, and much more.
 
-If you prefer the command line to a Cloud SQL editor, use [psql][install-psql].
+- [SQL editor][run-sqleditor]: a simple UI in ops mode that you use to run ad-hoc ephemeral 
+  queries. This is useful for quick one-off tasks like creating an index on a small table or 
+  inspecting pg_stat_statements.
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
  inspecting `pg_stat_statements`.
```

File: getting-started/run-queries-from-console.md [hunk 11]
```diff
@@ -9,38 +9,42 @@ content_group: Getting started
...
+
+
+If you prefer the command line to ops mode SQL editors in Timescale Console, use [psql][install-psql].
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
If you prefer the command line to the ops mode SQL editor in Timescale Console, use [psql][install-psql].
```

File: getting-started/run-queries-from-console.md [hunk 12]
```diff
@@ -9,38 +9,42 @@ content_group: Getting started
...
 
 ## Data mode
 
-You use `Data` in Timescale Console to write queries, visualize data, and share your results. 
+You use data mode in Timescale Console to write queries, visualize data, and share your results.
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
You use the data mode in Timescale Console to write queries, visualize data, and share your results. 
```

File: getting-started/run-queries-from-console.md [hunk 13]
```diff
@@ -9,38 +9,42 @@ content_group: Getting started
...
-- **[Charts](https://docs.popsql.com/docs/creating-charts)**: visualize data from inside your SQL editor rather than switch to Sheets or Excel.
+- **[Charts](https://docs.popsql.com/docs/creating-charts)**: visualize data from inside the UI rather than switch to Sheets or Excel.
 - **[Schedules](https://docs.popsql.com/docs/scheduled-queries)**: automatically refresh queries and dashboards to create push alerts.
 - **[Query variables](https://docs.popsql.com/docs/query-variables)**: use Liquid to parameterize your queries or use `if` statements.
 - **Cross platform**:  works from [Timescale Console][portal-data-mode] or download the 
   [desktop](https://popsql.com/download) app for macOS, Windows, and Linux.
 - **Easy connection**: to Timescale Cloud, PostgreSQL, Redshift, Snowflake, BigQuery, MySQL, SQL Server, [and more](https://popsql.com/connections).
 
-### Connect to your Timescale Cloud service in Data mode
+### Connect to your Timescale Cloud service in data mode
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
### Connect to your Timescale Cloud service in the data mode
```

File: getting-started/run-queries-from-console.md [hunk 14]
```diff
@@ -54,7 +58,7 @@ To connect to a service:
 
 1. **Connect to your service**
 
-   1. In [`Data` in Timescale Console][portal-data-mode], select a service and enter your password, then click `Connect`.
+   1. In [data mode][portal-data-mode] in Timescale Console, select a service and enter your password, then click `Connect`.
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
   1. In the [data mode][portal-data-mode] in Timescale Console, select a service and enter your password, then click **Connect**.
```

File: getting-started/run-queries-from-console.md [hunk 15]
```diff
@@ -68,14 +72,14 @@ To connect to a service:
 
 
 Quick recap. You:
-- Manage your services in the [`Ops` in Timescale Console][portal-ops-mode]
-- Manage your data in [`Data` in Timescale Console][portal-data-mode]
+- Manage your services in the [ops mode in Timescale Console][portal-ops-mode]
+- Manage your data in [data mode in Timescale Console][portal-data-mode]
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
- Manage your data in the [data mode in Timescale Console][portal-data-mode]
```

File: getting-started/run-queries-from-console.md [hunk 16]
```diff
@@ -68,14 +72,14 @@ To connect to a service:
...
 - Store configuration and security information in your config file.
 
 </Procedure>
 
 
-Now you have used Data mode in Timescale Console, see how to easily do the following:
+Now you have used data mode in Timescale Console, see how to easily do the following:
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
Now you have used the data mode in Timescale Console, see how to easily do the following:
```

File: getting-started/run-queries-from-console.md [hunk 17]
```diff
@@ -100,12 +104,12 @@ If your Timescale Service runs inside a VPC, do one of the following to enable a
   - When you configure the connection in PopSQL, under `Advanced Options`, enable `Connect over SSH`
 - Add PopSQL's static IPs (`23.20.131.72, 54.211.234.135`) to your allowlist
 
-#### What happens if another member of my Timescale project Data mode?
+#### What happens if another member of my Timescale project data mode?
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]: enter/uses the data mode?

File: getting-started/run-queries-from-console.md [hunk 18]
```diff
@@ -100,12 +104,12 @@ If your Timescale Service runs inside a VPC, do one of the following to enable a
...
 
-The number of Data mode seats you are allocated depends on your
+The number of data mode seats you are allocated depends on your
 [Pricing Plan][pricing-plan-features].
 
-#### Will using Data mode affect the performance of my Timescale Cloud service?
+#### Will using data mode affect the performance of my Timescale Cloud service?
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
#### Will using the data mode affect the performance of my Timescale Cloud service?
```

File: getting-started/run-queries-from-console.md [hunk 19]
```diff
@@ -121,11 +125,11 @@ a [read replica][readreplica] and send analytical queries there.
 
 If you'd like to prevent write operations such as insert or update. Instead 
 of using the `tsdbadmin` superuser, create a read-only user for your service and
-use that in Data mode.
+use that in data mode.
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
use that in the data mode.
```

File: getting-started/run-queries-from-console.md [hunk 20]
```diff
@@ -135,15 +139,15 @@ for an Timescale Cloud service.
 To enable or disable SQL editor in your service, click `Operations` > `Service management`, then
 update the setting for SQL editor.
 
-### Run queries from Console with SQL editor
+### Run queries from ops mode in Timescale Console
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
### Run queries from the ops mode in Timescale Console
```

File: getting-started/run-queries-from-console.md [hunk 21]
```diff
@@ -135,15 +139,15 @@ for an Timescale Cloud service.
...
 
 <Procedure>
 
 To use SQL editor with Timescale:
 
 1.  **Open SQL editor from Timescale Console**
 
-    In [`Ops` in Timescale Console][portal-ops-mode], select a service, then click `SQL editor`.
+    In [Ops mode][portal-ops-mode] in Timescale Console, select a service, then click `SQL editor`.
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
    In the [ops mode][portal-ops-mode] in Timescale Console, select a service, then click **SQL editor**.
```

File: getting-started/run-queries-from-console.md [hunk 22]
```diff
@@ -156,9 +160,8 @@ To use SQL editor with Timescale:
 
 ## Cloud SQL editor licenses
 
-* **SQL editor**: free for anyone with a [Timescale Cloud account][create-cloud-account].
-* **Data mode with Timescale Cloud**: the number of Cloud SQL editor seats you are allocated depends on your 
-  [Pricing Plan][pricing-plan-features].
+* **SQL editor in ops mode**: free for anyone with a [Timescale Cloud account][create-cloud-account].
```
Comment (atovpeko) [inline getting-started/run-queries-from-console.md]:
```suggestion
* **SQL editor in the ops mode**: free for anyone with a [Timescale Cloud account][create-cloud-account].
```

**Review by atovpeko — CHANGES_REQUESTED** [review]
A lot of very minor things

**Review by atovpeko — APPROVED** [review]

File: getting-started/run-queries-from-console.md [hunk 23]
```diff
@@ -9,38 +9,42 @@ content_group: Getting started
...
-- [Data mode][run-popsql]: a comprehensive UI that you use to write queries, visualize data, 
-   and share your results.
-- [SQL editor][run-sqleditor]: a simple UI to run queries for each individual service.
+In Timescale Console you can use the following ways to run SQL queries against your database:
 
+- [Data mode][run-popsql]: a rich experience powered by PopSQL. You can write queries with 
+  autocomplete, save them in folders, share them, create charts/dashboards, and much more.
 
-If you prefer the command line to a Cloud SQL editor, use [psql][install-psql].
+- [SQL editor][run-sqleditor]: a simple UI in the ops mode that you use to run ad-hoc ephemeral
```
Comment (rahilsondhi) [inline getting-started/run-queries-from-console.md]:
```suggestion
- [SQL editor in Ops mode][run-sqleditor]: a simple SQL editor in Ops mode that lets you run ad-hoc ephemeral 
```

**Pull request merged**